server/channels/app/export.go

```diff
@@ -182,15 +195,18 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 		return appErr
 	}
 
-	ctx.Logger().Info("Bulk export: exporting direct channels")
-	if appErr = a.exportAllDirectChannels(ctx, job, writer, opts.IncludeArchivedChannels); appErr != nil {
-		return appErr
-	}
+	var directAttachments []imports.AttachmentImportData
+	if teamId == "" {
```

Comment:
Let's add a log message for the single teamID case that makes it clear that no DMs are getting exported